docs/interactive_learning.rst

```diff
@@ -171,14 +200,14 @@ a ``form:`` prefix in the generated story. For example:
         - slot{"requested_slot": null}
         - utter_slots_values
 
+
 Input validation
 ~~~~~~~~~~~~~~~~
 
-Every time the user enters some unexpected input (i.e. doesn't provide the
-requested slot), you will be asked whether you want the form action to validate
-the users input when returning to it. This is necessary because in certain
-situations you may not want the users current input to be used to fill the
-current requested slot. Take the example below:
+Every time the user responds with something *other* than the requested slot, 
```

Comment:
actually sorry this was a mistake on my part, it should be other than the requested slot or any of the other required slots